snippets/base/models.py

```diff
@@ -2083,13 +2081,15 @@ def update_asrsnippet_modified_date(sender, instance, **kwargs):
 
     elif (isinstance(instance, Campaign) or
```

Comment:
NIT: I realize you aren't changing this, but this is equivalent to: `isinstance(instance, (Campaign, Target)):`

Reply:
Didn't know that, thanks!